modules/packages/LinearAlgebra.chpl

```diff
@@ -1883,8 +1883,32 @@ module Sparse {
     return B;
   }
 
-  /* Transpose CSR matrix */
-  proc _array.T where isCSArr(this) { return transpose(this); }
+  /* Transpose Sparse matrix, supports CSR and COO. */
+  proc _array.T where isSparseArr(this) { return transpose(this); }
+
+  pragma "no doc"
+  proc transpose(D: domain) where isSparseDom(D) && !isCSDom(D) {
```

Comment:
See the current implementation of the other transpose overload for CSR domains for an easier way to implement this: https://github.com/chapel-lang/chapel/blob/master/modules/packages/LinearAlgebra.chpl#L2181

Reply:
Could you explain more why is it easier in the example you gave?

Reply:
I was referring the [buffered index additions to sparse domains](https://chapel-lang.org/docs/master/builtins/ChapelArray.html#ChapelArray.makeIndexBuffer) that had been recently added.